gradle/mockito-junit-jupiter/javadoc.gradle

```diff
@@ -34,6 +34,12 @@ tasks.named("javadoc", Javadoc) {
     options.use = true
     options.windowTitle = "Mockito JUnit Jupiter ${project.version} API"
 
+    // https://bugs.openjdk.org/browse/JDK-8274639
+    if (JavaVersion.current() >= JavaVersion.VERSION_18) {
+        options.addStringOption('-link-modularity-mismatch', 'info')
+        options.links("https://junit.org/junit5/docs/5.11.0/api/")
```

Comment:
Ideally, the JUnit version number shouldn't be hardcoded.

I see it appears here:

https://github.com/mockito/mockito/blob/f6450a01f27d0e4e7e95a52025b000726558774d/gradle/dependencies.gradle#L8

What would be the proper way to expose that value to this file?

Reply:
If I understand correctly, we need to define the `versions` to be exposed on the project as well. For that, we can do the following:

```groovy
ext {
    libraries = [:]
    libraryVersions = [:]
}

libraryVersions.bytebuddy = '1.15.0'
libraryVersions.junitJupiter = '5.11.0'
libraryVersions.errorprone = '2.23.0'
```

And then here this should work:

```groovy
options.links("https://junit.org/junit5/docs/${libraryVersions.junitJupiter}/api/")
```

Reply:
Solved with the introduction of [`libs.versions.toml`](https://docs.gradle.org/current/userguide/platforms.html#sub:conventional-dependencies-toml).